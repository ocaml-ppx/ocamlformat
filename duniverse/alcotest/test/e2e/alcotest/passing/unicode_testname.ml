let () =
  Alcotest.run
    "Suite name containing file separators / and non-ASCII characters ðŸ”¥"
    [
      ( "ðŸ”¥",
        [
          Alcotest.test_case "Non ASCII unicode character" `Quick (fun () -> ());
        ] );
      ( "ðŸ”¥a-b",
        [
          Alcotest.test_case "Non ASCII and ASCII characters" `Quick (fun () ->
              ());
        ] );
    ]
