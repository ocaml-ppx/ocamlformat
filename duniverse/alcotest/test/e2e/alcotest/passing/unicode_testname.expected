Testing `Suite name containing file separators / and non-ASCII characters ğŸ”¥'.
This run has ID `<uuid>'.

  [OK]          ğŸ”¥             0   Non ASCII unicode character.
  [OK]          ğŸ”¥a-b          0   Non ASCII and ASCII characters.

Full test results in `<build-context>/_build/_tests/<test-dir>'.
Test Successful in <test-duration>s. 2 tests run.
